dlc_developer_config.toml

```diff
@@ -35,6 +35,6 @@ sagemaker_local_tests = false
 # "off" --> do not trigger sagemaker remote tests (default)
 # "standard" --> run standard sagemaker remote tests from test/sagemaker_tests
 # "rc" --> run release_candidate_integration tests
-sagemaker_remote_tests = "off"
+sagemaker_remote_tests = "rc"
```

Comment:
looks like rc tests pass - feel free to switch this to "standard" and we can debug any issues with standard suite of SM remote tests